Dockerfile.mme

```diff
@@ -0,0 +1,340 @@
+################################################################
```

Comment:
This also does not belong in the root directory. Can you create a lte/gateway/docker directory and move this file there?

Reply:
fixed